plugins/module_utils/network/ios/rm_templates/interfaces.py

```diff
@@ -129,5 +129,19 @@ def __init__(self, lines=None, module=None):
                 },
             },
         },
+        {
+            "name": "template",
+            "getval": re.compile(
```

Comment:
It would be good to have at least 1 integration test for this

Reply:
Added test code for each state type